lib/ae_mdw/db/model.ex

```diff
@@ -223,12 +223,10 @@ defmodule AeMdw.Db.Model do
 
   # AEX9 balance:
   #     index = {contract_pk, account_pk}
-  #     block_index = {kbi, mbi}
   #     amounts: float
   @type aex9_balance ::
           record(:aex9_balance,
             index: {Db.pubkey(), Db.pubkey()},
-            block_index: Blocks.block_index(),
```

Comment:
:+1: nice workaround to avoid having to redefine the structure, but at some point we should probably change it, perhaps we can create a GH issue?

Reply:
It was removed because it's not being used on fetch (e.g the history uses the contract balance instead).

Reply:
I understand that, but it wasn't removed from the database, every time we store this record we are storing 3 values unnecessarily. This is what I think we should work on in the future

Reply:
Humm, I see what you mean. This is a change that demands to start a sync from scratch.

Reply:
If we remove it from the default without a sync from scratch it will break the existing records. We can either:

1. leave it on the default and perform the change in a migration (which we can add later on)
2. remove the default and require a sync from scratch

I'm fine either way, but perhaps we can do 1. for now to avoid syncing from scratch?

Reply:
I will provide a migration, thanks. There is a another fix needed for contract creation, drafting ...